response_operations_ui/templates/edit-collection-exercise-details.html

```diff
@@ -59,12 +59,14 @@ <h1 class="jupiter">Edit collection exercise details</h1>
 
                   <br>
 
+                {% if ce_state != 'READY_FOR_LIVE' and ce_state != 'LIVE' %}
```

Comment:
I put this in flask jinja so that if the state was `ready for live` or `live` then the option to edit the period for the user disappeared. Does this need to be added in the view too?

Reply:
It's not needed in addition to the template code, but it would be better to have the logic related to states instead in the view to keep the business logic in one place.

e.g. [something here perhaps](https://github.com/ONSdigital/response-operations-ui/blob/e234d61bf651e660b78acabc6c616c7fdcaf1e49/response_operations_ui/views/collection_exercise.py#L228): 

```python
def view_collection_exercise_details(short_name, period):
...
    ce_state = ce_details['collection_exercise']['state']
    show_edit_period = ce_state not in ('READY_FOR_LIVE', 'LIVE')
...
    return render_template('edit-collection-exercise-details.html', show_edit_period=show_edit_period, ...
```

then in the template:

```html
...
{% if show_edit_period %}
...
{% endif %}
...
```

Reply:
Oh I understand yeah thats a lot cleaner way to implement that, thank you :)